### PR TITLE
project start

### PR DESCRIPTION
**Pull request by LarynaB**

### CODE REVIEW

File: src/iterative_sorting/iterative_sorting.py [hunk 1]
```diff
@@ -7,15 +7,16 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for small in range(cur_index, len(arr)):
+            if arr[small] <  arr[smallest_index]:
```
Comment (tryingtokeepup) [inline src/iterative_sorting/iterative_sorting.py:11]: Excellent job with your selection sort!

**Review by tryingtokeepup — COMMENTED** [review]
Were you able to complete any other sorts?

**Comment by tryingtokeepup** [comment]
You passed the sprint, so job well done, but see if you can't revisit this project in the future. Trust me, these sorting algos will be useful in the future! :D

**Comment by LarynaB** [comment]
Ok will get to back to them eventually. Having ADD on what to focus on at the moment.

________________________________

**Pull request merged**